src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -101,14 +105,20 @@ def report_initial_status(
         )
 
 
-def try_run_vipin(log_path: Path) -> None:
+def try_run_vipin(output_paths: OutputPaths) -> None:
+    log_path = output_paths.worker_logging_root
     try:
-        report_performance(
+        perf_df = report_performance(
             input_directory=log_path, output_directory=log_path, output_hdf=False, verbose=1
         )
     except Exception as e:
         logger.warning(f"Performance reporting failed with: {e}")
 
```

Comment:
maybe add a `return` here? if creating `perf_df` fails, then certainly appending to the central logs would fail too